### PR TITLE
Fix cpu-adam AVX performance

### PR DESCRIPTION
**Pull request by RezaYazdaniAminabadi**
This fixes https://github.com/microsoft/DeepSpeed/issues/1635

### CODE REVIEW

File: deepspeed/ops/transformer/inference/transformer_inference.py [hunk 1]
```diff
@@ -146,6 +146,31 @@ def _transpose_for_context(x):
                                       (hidden_size_per_partition,)
             return x.view(*new_x_layer_shape)
 
+        def backup_attention(mixed_query, key_layer, value_layer, input_mask):
```
Comment (jeffra) [inline deepspeed/ops/transformer/inference/transformer_inference.py:149]: @RezaYazdaniAminabadi, was this an accidental addition? seems unrelated to the PR right?

**Review by jeffra — APPROVED** [review]

File: deepspeed/ops/transformer/inference/transformer_inference.py (hunk 1 above)
Comment (RezaYazdaniAminabadi) [reply deepspeed/ops/transformer/inference/transformer_inference.py:149]: Thanks Jeff, will remove it!
